src/base.jl

```diff
@@ -178,7 +178,7 @@ function ConcreteSystem(sys::System)
         end
     end
 
-    loadZones = get(sys.annex, :LoadZones, nothing)
+    !isnothing(sys.annex) ? loadZones = get(sys.annex, :LoadZones, nothing) : loadZones = nothing
```

Comment:
How about this instead?  (also noticed that I changed loadZones to load_zones)

load_zones = isnothing(sys.annex) ? nothing : get(sys.annex, :LoadZones, nothing)

Reply:
The change looks good to me. I haven't seen the change in load zone naming.

Reply:
I was just referring to this local variable.  Our convention is snake_case.